app/controllers/standup_meeting_groups/joins_controller.rb

```diff
@@ -7,12 +8,8 @@ def create
 
     authorize @standup_meeting_group_user, policy_class: StandupMeetingGroup::JoinPolicy
 
-    @standup_meeting_group_user.save!
-
-    respond_to do |format|
-      format.turbo_stream do
-        flash.now[:notice] = "You have joined #{@standup_meeting_group.name}!"
-      end
```

Comment:
The `respond_to` block has been removed since
1. A request received at this controller action’s route is automatically accepted as a `turbo_stream` one.
2. We’re not handling any other formats.